Make sure you have the requirements installed.

To see this bug, run `python server.py` in one process.
Then, open a new terminal and run `python client.py`. 

See the print statements and comments in server.py for 
my observations.
